tsconfig.json

```diff
@@ -3,7 +3,7 @@
       /* Basic Options */
       "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
       "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-      "lib": ["ES2015"],                        /* Specify library files to be included in the compilation. */
+      "lib": ["DOM", "ES2015"],                 /* Specify library files to be included in the compilation. */
```

Comment:
noticed you added 'DOM' here, is this intentional?

Reply:
The `console` global is defined in the DOM lib.  I'm using the console to emit warnings when using the deprecated properties.

Reply:
I see... based on tsconfig [doc](https://www.typescriptlang.org/docs/handbook/compiler-options.html):

> Note: If --lib is not specified a default list of libraries are injected. The default libraries injected are:
► For --target ES5: DOM,ES5,ScriptHost
► For --target ES6: DOM,ES6,DOM.Iterable,ScriptHost

seems like target `es6` by default already include DOM, in addition to `es2015`, maybe we don't need to specify `lib` at all?

Anyway, it's not wrong just a bit redundant...

Reply:
I'm fine with that.  Just commented the line to match the rest of the config file.  We'll consolidate that config once everything is converted to TS.